kroxylicious-app/pom.xml

```diff
@@ -176,6 +176,7 @@
                             <descriptors>
                                 <descriptor>src/assembly/binary-distribution.xml</descriptor>
                             </descriptors>
+                            <tarLongFileMode>posix</tarLongFileMode>
```

Comment:
no objections but probably really ought to be in a separate PR.

Reply:
Yeah, I agree. It was failing for me and I changed it. I can restore it and then create another PR.